src/aosm/azext_aosm/tests/latest/integration_tests/integration_test_mocks/mock_input_templates/input_multi_nf_nsd.jsonc

```diff
@@ -0,0 +1,52 @@
+{
+  "location": "uksouth",
```

Comment:
I wonder if we should remove this test, because i dont think it actaully works at the sns level?